aspnetcore/security/authentication/social/google-logins.md

```diff
@@ -53,38 +53,27 @@ You can manage your API credentials and usage in the [API Console](https://conso
 
 ## Configure Google authentication
 
-Add the [`Microsoft.AspNetCore.Authentication.Google`](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Google) NuGet package to the app.
-
-:::moniker range="< aspnetcore-6.0"
-
-Add the Authentication service to the `Startup.ConfigureServices`:
-
-[!code-csharp[](~/security/authentication/social/social-code/3.x/StartupGoogle3x.cs?highlight=11-19)]
-
-:::moniker-end
-
-:::moniker range=">= aspnetcore-6.0"
-
-Add the Authentication service to the `Program`:
-
-[!code-csharp[](~/security/authentication/social/social-code/6.x/ProgramGoogle.cs)]
-
-:::moniker-end
+* Add the [`Google.Apis.Auth.AspNetCore3`](https://www.nuget.org/packages/Google.Apis.Auth.AspNetCore3) NuGet package to the app.
+* Add the Authentication service to the `program.cs`:
+* Follow [`Add Authtication for asp.net app`](https://developers.google.com/api-client-library/dotnet/guide/aaa_oauth#configure-your-application-to-use-google.apis.auth.aspnetcore3)
 
 [!INCLUDE [default settings configuration](includes/default-settings2-2.md)]
 
 ## Sign in with Google
+* Get a link to the libary at [google developer library link ](https://developers.google.com/identity/gsi/web/guides/client-library) to get link of library.
+* Then go to [google developer button genration ](https://developers.google.com/identity/gsi/web/tools/configurator)
+* Setup your Controller to match with ` data-login_uri="{HostName}/{ControllerName}/{actionName}" ` attrbute because after success login it will forward you to that link.
+* Create controller and action takes one argement  `string credential` because that what google return when complete login process.
+* Verify `credential` by using this line of code
+`GoogleJsonWebSignature.Payload payload = await GoogleJsonWebSignature.ValidateAsync(credential);`
+* Here you  get all information about login user you can store it in database.
 
-* Run the app and select **Log in**. An option to sign in with Google appears.
-* Select the **Google** button, which redirects to Google for authentication.
-* After entering your Google credentials, you are redirected back to the web site.
-
-[!INCLUDE[Forward request information when behind a proxy or load balancer section](includes/forwarded-headers-middleware.md)]
-
-[!INCLUDE[](includes/chain-auth-providers.md)]
+[Complete Simple working app](https://github.com/sharafabacery/GoogleAuthticationExample)
```

Comment:
See https://github.com/dotnet/AspNetCore.Docs.Samples/issues/264

Reply:
I add the sample